app/assets/src/components/common/Header.jsx

```diff
@@ -172,7 +188,7 @@ const UserMenuDropDown = ({
         }
       />,
       <BareDropdown.Item
-        key="5"
+        key="6"
```

Comment:
this was there before and you don't need to change it, but for your future reference keys should be more unique https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-array-index-key.md